art/art_dic.py

```diff
@@ -64,9 +64,9 @@
                 "cat1": "=^..^=",
                 "fish swim": "¸.·´¯`·.´¯`·.¸¸.·´¯`·.¸><(((º>",
                 "sword1": "(===||:::::::::::::::>",
-                "rock on1": "\,,/(^_^)\,,/",
-                "rock on2": "\m/(-_-)\m/",
-                "caterpillar": ",/\,/\,/\,/\,/\,/\,o",
+                "rock on1": r"\,,/(^_^)\,,/",
+                "rock on2": r"\m/(-_-)\m/",
+                "caterpillar": r",/\,/\,/\,/\,/\,/\,o",
```

Comment:
pep8 has added the `r`s to code, should i change the configuration of pep8 @sepandhaghighi ?

Reply:
You should ignore `W605` in `autopep8`. (Take a look at [autopep8 repo](https://github.com/hhatto/autopep8#usage))
This error support added in new version of `autopep8` that releases about 25days ago.

Visit [here](https://www.flake8rules.com/rules/W605.html) for more information about this error.

Reply:
should we add this flag to `autopep8.sh` and `autopep8.bat` so that this won't happen?
like:
```
python -m autopep8 art --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose --ignore w605
```

Reply:
> should we add this flag to `autopep8.sh` and `autopep8.bat` so that this won't happen?
> like:
> 
> ```
> python -m autopep8 art --recursive --aggressive --aggressive --in-place --pep8-passes 2000 --verbose --ignore w605
> ```

Yeah, `autopep9.sh` and `autopep8.bat` should be updated.